packages/notification/style.module.css

```diff
@@ -114,6 +114,18 @@
   }
 }
 
+.heading {
+  --color: var(--wpl-neutral-700);
+
+  composes: g-type-body-strong from global;
+  margin: 0;
+  color: var(--color);
+
+  @nest .dark & {
+    color: var(--wpl-neutral-0);
```

Comment:
Very non-blocking ⛏️ : was there a specific motivation behind assigning a new value to `color` vs the `--color` custom property here? Curious!

Reply:
Good catch! updated.